universe/Condition.cpp

```diff
@@ -913,7 +913,7 @@ bool SortedNumberOf::SourceInvariant() const
           (!m_condition || m_condition->SourceInvariant())); }
 
 std::string SortedNumberOf::Description(bool negated/* = false*/) const {
-    std::string number_str = m_number->ConstantExpr() ? boost::lexical_cast<std::string>(m_number->Dump()) : m_number->Description();
+    std::string number_str = m_number->ConstantExpr() ? std::to_string(m_number->Dump()) : m_number->Description();
```

Comment:
`m_number->Dump()` always returns a `std::string` and there exists no `std::to_string` variant that accepts a `std::string`.